packages/vuetify/src/components/VCalendar/VCalendar.ts

```diff
@@ -335,9 +345,10 @@ export default CalendarWithEvents.extend({
           }
         }
 
-        categories = Object.keys(categoryMap)
+        categories = categories.filter(v => {
```

Comment:
can be one-lined

Reply:
Not sure I understand this one

Reply:
`categories = categories.filter(v => Object.keys(categoryMap).includes(v.categoryName))`